i18n/en-US.yml

```diff
@@ -857,7 +857,7 @@ util:
     networkUnavailable: The {network} network is unavailable at this moment.
     noTripFound: No trip found.
     noTripFoundForMode: No trip found for {modes}.
-    noTripFoundReason: There may be no transit service within the maximum specified distance or at the specified time, or your start or end point might not be safely accessible.
+    noTripFoundReason: There may be no transit service within the maximum specified distance or at the specified time, or your start or end point might not be safely accessible. <strong>An itinerary may be found if you adjust some Transit Options & Preferences above.</strong>
```

Comment:
Place the <strong> tag in a separate line.

Reply:
How about replacing "An itinerary may be found..." with "Try adjusting your search options"? (Careful, in batch routing, there is no "Transit Options & Preferences" text, and there is no "above" in mobile (you have to click Edit to change the search).

Reply:
How about "Try editing your search preferences" that covers all cases -- preferences hints towards the non-batch mode, edit hints towards the mobile edit button, and search preferences hints towards the batch mode gear

Reply:
That works for me.

Reply:
Yes the strong text was requested by users

Reply:
Removed the hardcoded string update in a26a1d74f96c8a675361ddf0b58bb0e4b3c9323a